packages/design/src/utils/match/index.ts

```diff
@@ -0,0 +1,6 @@
+import isMatch, { MatchCallback } from './match';
+import { dateMatcher, dateTimeMatcher } from './matchers';
+
+export default isMatch;
+export { dateMatcher, dateTimeMatcher };
+export type { MatchCallback };
```

Comment:
what is `export type { MatchCallback };`?

```suggestion
export { dateMatcher, dateTimeMatcher, MatchCallback };
```

Reply:
Re-exporting the type this way causes this error:

```
export 'MatchCallback' (reexported as 'MatchCallback') was not found in './match' (possible exports: default)
design/src/utils/match/index.ts
```

We can use [type-only exports](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#-type-only-imports-and-export) to fix this

Reply:
ah got it thanks